public/app/features/auth-config/index.ts

```diff
@@ -21,13 +23,13 @@ export function getRegisteredAuthProviders(): AuthProviderInfo[] {
 }
 
 export function getAuthProviderInfo(provider: string) {
-  return registeredAuthProviders.find((p) => p.id === provider);
+  return registeredAuthProviders.find((p: AuthProviderInfo) => p.id === provider);
```

Comment:
I don't think type needs to be explicitly mentioned here, it's correctly evaluated as `AuthProviderInfo` implicitly from `registeredAuthProviders`